<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gvMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Essential Information:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tedtInfo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Workpieces:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWorkpiece">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSysRun">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>System.run</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHello"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionSelect_Camare"/>
    <addaction name="actionDetect"/>
    <addaction name="actionScreen_shot"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuCameraViewport">
     <property name="title">
      <string>CameraViewport</string>
     </property>
     <addaction name="actionZoon_in"/>
     <addaction name="actionZoom_out"/>
    </widget>
    <addaction name="menuCameraViewport"/>
    <addaction name="separator"/>
    <addaction name="actionCriterionList"/>
    <addaction name="actionCiterionEditor"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSystem"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCriterionList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CriterionList</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionSelect_Camare">
   <property name="text">
    <string>Select Camera</string>
   </property>
  </action>
  <action name="actionCiterionEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CriterionEditor</string>
   </property>
  </action>
  <action name="actionZoon_in">
   <property name="text">
    <string>zoon in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>zoom out</string>
   </property>
  </action>
  <action name="actionDetect">
   <property name="text">
    <string>Detect</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
  <action name="actionScreen_shot">
   <property name="text">
    <string>Screen shot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
